### ISSUE TITLE
Create directory structure

### ISSUE THREAD

**Issue by Sha-nks** [fxsth/pledo#7]
Hello, i really like your project, but i have detected a problem for me.

1. Is it normal, that i can only choose drive letter C; to save my files?

and 

2. Is it possible to create for each film a Directory and for each show a directory, season directory and episode directory?

Thanks for telling us your great project.

**Comment by fxsth** [comment]
Hi,
1) no that seems to be a bug. Which platform are you using? Windows, Linux, Docker or desktop?
2) yes, that should be possible. Ill try to find some time to add this as a setting.

**Comment by Sha-nks** [comment]
I'm Using the Windows build .exe application.

**Comment by fxsth** [comment]
Should now be fixed with release v1.0.1 and added settings for different sub download directories (movie, season, etc).

**Comment by alafad** [comment]
Thanks for your Update, after updating my docker image to master or latest, the container stopps with the following output:
```
The command could not be loaded, possibly because:

  * You intended to execute a .NET application:

      The application 'Web.dll' does not exist.

  * You intended to execute a .NET SDK command:

      No .NET SDKs were found.

Download a .NET SDK:

https://aka.ms/dotnet-download

Learn about SDK resolution:

https://aka.ms/dotnet/sdk-not-found
```

What could be the problem?

**Comment by fxsth** [comment]
Sorry, I broke the dockerfile. Now fixed in v1.0.2 (latest).
